Src/StdLib/Lib/test/test_ensurepip.py

```diff
@@ -38,11 +38,19 @@ class TestBootstrap(EnsurepipMixin, unittest.TestCase):
     def test_basic_bootstrapping(self):
         ensurepip.bootstrap()
 
+        if sys.platform == 'cli':
+            args = [
+                    "install", "--no-index", "--find-links",
+                    mock.ANY, "--no-compile", "setuptools", "pip",
```

Comment:
So we're just adding no-compile everywhere for the cli? Does the args call work with an empty string? Maybe we could be slightly less verbose:
```
noCompileIfCli = "--no-compile" is sys.platform == "cli" else "" # this could be a global
[
    "install", "--no-index", "--find-links",
    mock.ANY, noCompileIfCli, "setuptools", "pip",
]
```
Anyway, just a thought...

Reply:
This doesn't work on C python, it fails every test because of the difference in argument lists.

Reply:
Alright, we can keep it as is.